R/board_azure.R

```diff
@@ -145,38 +179,83 @@ board_deparse.pins_board_azure <- function(board, ...) {
   url <- paste0(container$endpoint$url, container$name)
 
   container <- expr(AzureStor::storage_container(!!url))
-  expr(board_azure(!!container))
+  expr(board_azure(!!container, path = !!board$path))
 }
 
 
 # Helpers -----------------------------------------------------------------
 
 azure_delete_dir <- function(board, dir) {
-  ls <- AzureStor::list_storage_files(board$container, dir)
-  for (path in ls$name) {
-    AzureStor::delete_storage_file(board$container, path, confirm = FALSE)
+  dir <- azure_normalize_path(board, dir)
+
```

Comment:
Maybe add comment with link to corresponding AzureStor issue if it exists? (Just to remind us to eventually simplify this code once the logic is in AzureStor)

Reply:
Not sure which bit you're referring to here, can you clarify?

Reply:
I thought in your other comment you were implying that this complicated delete code would eventually live in AzureStore?

Reply:
Oh right. Yeah, the delete code is actually implemented in AzureStor now, as of a couple months ago. I'll add a comment to this effect.